fedn/cli/login_cmd.py

```diff
@@ -0,0 +1,58 @@
+import yaml
+import click
+import requests
+
+from .main import main
+from getpass import getpass
+
+# Replace this with the platform's actual login endpoint
+URL = "https://fedn.scaleoutsystems.com/api/token/"
```

Comment:
**Cool stuff!** This one should be configurable by the user. Perhaps default to _https://fedn.scaleoutsystems.com_ (and then concat the url programatically). This is because FEDn Studio can be hosted on different domains and we would like this to work on all instances.